constants.js

```diff
@@ -0,0 +1,38 @@
+export const NETWORKS = {
+  arbitrumGoerli: {
```

Comment:
Moved all these values to a constant file so they didn't take up room in the demo file